### PR TITLE
PIM-5900: Fix login CSS for big buttons

### PR DESCRIPTION
**Pull request by pierallard**
It fixes bad display of the buttons, using a flex solution for the inputs.

### CODE REVIEW

File: src/Pim/Bundle/UserBundle/Resources/views/Security/login.html.twig [hunk 1]
```diff
@@ -27,8 +27,8 @@
     <form action="{{ path('oro_user_security_check') }}" method="post" class="form-signin form-inline">
         <fieldset>
             <div class="form-row">
-                <input type="text" id="prependedInput" autofocus="autofocus" name="_username" value="{{ last_username }}" required="required" placeholder="{{ 'Username or Email'|trans }}" />
-                <input type="password" id="prependedInput2" name="_password" required="required" placeholder="{{ 'Password'|trans }}" autocomplete="off" />
+                <input type="text" id="prependedInput" autofocus="autofocus" name="_username" value="{{ last_username }}" required="required" placeholder="{{ 'Username or Email'|trans }}" size="1"/>
```
Comment (phaseinducer) [inline src/Pim/Bundle/UserBundle/Resources/views/Security/login.html.twig]: I don't understand the impact of this

**Comment by phaseinducer** [comment]
Little comment & GTM

File: src/Pim/Bundle/UserBundle/Resources/views/Security/login.html.twig (hunk 1 above)
Comment (pierallard) [inline src/Pim/Bundle/UserBundle/Resources/views/Security/login.html.twig]: IRL!

**Comment by SamirBoulil** [comment]
Same comment as Yohan, can you explain in here ? Gtm then.

**Pull request merged**